src/Dfe.PlanTech.Domain/Core/Enums/Pages.cs

```diff
@@ -1,6 +1,9 @@
+
 namespace Dfe.PlanTech.Domain.Core.Enums;
 
-public enum Pages
+public static class Pages
 {
-    Landing
+    public const string Landing = "Landing";
+    
+    public const string SelfAssessment = "self-assessment";
```

Comment:
also, some inconsistency perhaps in uppercase/lowercase naming

Reply:
I would prefer to change "Landing" to "landing" (or "index", or something), however doing so would involve changing it in Contentful, and therefore break anyone elses copy until the changes are in.

Again, not ideal, so open for suggestions on changes here.